modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -28,17 +28,29 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 +
 [source,terminal]
 ----
-$ oc adm release info 4.11.0 --image-for=driver-toolkit
+# For x86 image:
```

Comment:
I think you need an explicit instruction to "run" or "enter" the command, and only one command per code block is allowed per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Maybe something like:

The image URL of the `driver-toolkit` corresponding to a certain release can be extracted from the release image using the `oc adm` command. 

For an x86 image, enter the following command:

`[example]`

For an ARM image, enter the following command:

`[example]`